core/bin/external_node/src/tests.rs

```diff
@@ -264,6 +265,7 @@ async fn node_reacts_to_stop_signal_during_initial_reorg_detection() {
     let opt = Cli {
         enable_consensus: false,
         components: "core".parse().unwrap(),
+        use_node_framework: false,
```

Comment:
Do we plan to add some test with  `use_node_framework: true`?

Reply:
We plan to get rid of this option and make the node framework to be used by default, so, ugh, in a way - yes.